index.bs

```diff
@@ -238,12 +244,18 @@ To <dfn>sanitize an element</dfn> named |element|, run these steps:
 
 1. let |config| be the |sanitizer|'s [=configuration=] dictionary.
 2. let |name| be |element|'s tag name.
-3. if |name| is contained in the built-in [=default element drop list=] return 'drop'.
-4. if |name| is in |config|'s [=element drop list=] return 'drop'.
-5. if |name| is contained in the built-in [=default element block list=] return 'block'.
-6. if |name| is in |config|'s [=element block list=] return 'block'.
-7. if |config| has a non-empty [=element allow list=] and |name| is not in |config|'s [=element allow list=] return 'block'
-8. return 'keep'
+3. if |name| is a [=valid custom element name=] and if |config|'s
+   [=allow custom elements option=] is unset or set to anything other than `true`, return 'drop'.
+4. if |name| is contained in the built-in [=default element drop list=] return 'drop'.
+5. if |name| is in |config|'s [=element drop list=] return 'drop'.
```

Comment:
I think it would be nice if we could run an initialization algorithm that combines a built-in list with a config list and then we'd just checks against that list during sanitization. Happy to have that happen in a follow up though :)

Reply:
Hm. Why?

I had thought about this early on and had initially tried to write it that way. But I don't think it's that easy to cleanly spec it that way, and some of the complexity will still remain. E.g., I don't think we can merge the "custom element name" condition into a list. If so, we'd end up with an algorithm that's split between two different places of the spec, which I suspect will be harder to understand since then the reader needs to connect the two parts on their own. So arguably, the current version is a bit clumsy, but it does strike me as easy to understand.